LINUX.md

```diff
@@ -18,22 +18,16 @@ Here's how to start:
  6. Install integration requirements
     - `sudo python setup.py install`
 
-2. **Set up your Mattermost outgoing webhooks**
- 1. Log in to your Mattermost account. Click the three dot menu at the top of the left-hand side and go to **Account Settings** > **Integrations** > **Outgoing Webhooks**
- 2. Under *Add a new outgoing webhook*, leave the *Channel* unselected and enter `gif:` into *Trigger Words*. You may select a channel if you only want this integration to be available in a specified channel
- 3. Paste your Web Server domain into *Callback URLs*, making sure to add `http://` to the beginning and `/new_post` to the end so it looks similar to `http://<your-web-server-domain>/new_post` and click **Add**
- 4. Copy the *Token* from your newly created webhook that appears under the *Existing outgoing webhooks* section
-
-3. **Set up your Mattermost slash command**
+2. **Set up your Mattermost slash command**
  1. Log in to your Mattermost account. Click the three dot menu at the top of the left-hand side and go to **Account Settings** > **Integrations** > **Slash Commands**
  2. Under *Add a new command*, enter `/gif` into **Command Trigger Word**
- 3. Paste your Web Server domain into *Callback URLs*, making sure to add `http://` to the beginning
+ 3. Paste your Web Server domain into *Callback URLs*, making sure to add `http://` to the beginning and `/new_post` to the end so it looks similar to `http://<your-web-server-domain>/new_post` and click **Add**
```

Comment:
My version of Mattermost (3.5.0) does not have **Add** at the stage mentioned above, just at the very end you need to click **Save**.
Also, the Add Slash Command page explicitly states "Trigger word must be unique, and **cannot begin with a slash** or contain any spaces." It adds the slash on its own.
Probably best to have the port number in the example, people may think the code adds it automatically from the environment variable.

> similar to _http://<your-web-server-domain>:<MATTERMOST_GIPHY_PORT>/new_post_